WcaOnRails/app/models/timestamp.rb

```diff
@@ -2,4 +2,8 @@
 
 class Timestamp < ApplicationRecord
   self.primary_key = "name"
+
+  def not_after?(other_date)
```

Comment:
So basically I started with `before?`, but quickly changed it to `never_or_before?` to make handling the `nil` case more verbose. As you can see Lars suggested two names that doesn't require the `never` part and still makes sense in the `nil` case. Does this make sense now? =)